saleor/discount/migrations/0072_promotionrule_variants_dirty.py

```diff
@@ -7,12 +7,26 @@ class Migration(migrations.Migration):
     dependencies = [
         ("discount", "0071_merge_20240307_1156"),
     ]
-
     operations = [
-        migrations.AddField(
-            model_name="promotionrule",
-            name="variants_dirty",
-            field=models.BooleanField(default=False),
+        migrations.SeparateDatabaseAndState(
+            database_operations=[
+                migrations.RunSQL(
+                    """
+                    ALTER TABLE discount_promotionrule
```

Comment:
Nothing important, but I was expecting to see the default definition in the field declaration, small improvement.

Reply:
Added, and ported to the rest of related PRs. Thanks